seeders/data/players.ts

```diff
@@ -38,6 +38,196 @@ const players: PLayerWithID[] = [
+    skill: 70,
+    position: PlayerPosition.striker,
+    status: PlayerStatus.starter,
+    fidelity: 100,
+    matchsPlayed: 1,
+  },
+  {
+    _id: new Types.ObjectId('63b1efd99726809eb241070a'),
+    firstName: 'Diego',
+    lastName: 'Castro',
+    skill: 70,
+    position: PlayerPosition.striker,
+    status: PlayerStatus.starter,
+    fidelity: 100,
+    matchsPlayed: 1,
+  },
+  {
+    _id: new Types.ObjectId('63b1efd99726809eb241070b'),
+    firstName: 'Julian',
+    lastName: 'Cedaro',
+    skill: 70,
+    position: PlayerPosition.striker,
+    status: PlayerStatus.starter,
+    fidelity: 100,
+    matchsPlayed: 1,
+  },
+  {
+    _id: new Types.ObjectId('63b1efd99726809eb241070c'),
+    firstName: 'Nicolas',
+    lastName: 'Blascow',
+    skill: 70,
+    position: PlayerPosition.striker,
+    status: PlayerStatus.starter,
+    fidelity: 100,
+    matchsPlayed: 1,
+  },
+  {
+    _id: new Types.ObjectId('63b1efd99726809eb241070d'),
+    firstName: 'Agustín',
+    lastName: 'Schiaffino',
+    skill: 70,
+    position: PlayerPosition.striker,
+    status: PlayerStatus.starter,
+    fidelity: 100,
+    matchsPlayed: 1,
+  },
+  {
+    _id: new Types.ObjectId('63b1efd99726809eb241070e'),
+    firstName: 'Rodrigo',
+    lastName: 'Sibrins',
+    skill: 70,
+    position: PlayerPosition.striker,
+    status: PlayerStatus.starter,
+    fidelity: 100,
+    matchsPlayed: 1,
+  },
+  {
+    _id: new Types.ObjectId('63b1efd99726809eb241070f'),
+    firstName: 'Martin',
+    lastName: 'Barea',
+    skill: 70,
+    position: PlayerPosition.striker,
+    status: PlayerStatus.starter,
+    fidelity: 100,
+    matchsPlayed: 1,
+  },
+  {
+    _id: new Types.ObjectId('63b1efd99726809eb2410710'),
+    firstName: 'Aquiles',
+    lastName: 'Mariani',
+    skill: 70,
+    position: PlayerPosition.striker,
+    status: PlayerStatus.starter,
+    fidelity: 100,
+    matchsPlayed: 1,
+  },
+  {
+    _id: new Types.ObjectId('63b1efd99726809eb2410711'),
+    firstName: 'Nicolas',
+    lastName: 'Viola',
+    skill: 70,
+    position: PlayerPosition.striker,
+    status: PlayerStatus.starter,
+    fidelity: 100,
+    matchsPlayed: 1,
+  },
+  {
+    _id: new Types.ObjectId('63b1efd99726809eb2410712'),
+    firstName: 'David',
+    lastName: 'Curras',
+    skill: 70,
+    position: PlayerPosition.striker,
+    status: PlayerStatus.starter,
+    fidelity: 100,
+    matchsPlayed: 1,
+  },
+  {
+    _id: new Types.ObjectId('63b1efd99726809eb2410713'),
+    firstName: 'Esteban',
+    lastName: 'Frare',
+    skill: 70,
+    position: PlayerPosition.striker,
+    status: PlayerStatus.starter,
+    fidelity: 100,
+    matchsPlayed: 1,
+  },
+  {
+    _id: new Types.ObjectId('63b1efd99726809eb2410714'),
+    firstName: 'Esteban',
+    lastName: 'Frare',
+    skill: 70,
+    position: PlayerPosition.striker,
+    status: PlayerStatus.starter,
+    fidelity: 100,
+    matchsPlayed: 1,
+  },
+  {
+    _id: new Types.ObjectId('63b1efd99726809eb2410714'),
```

Comment:
This document has the same `_id` as the one on line 212 (previous element), ids should be unique and in a hexadecimal format!